### DISCUSSION TITLE
Move all Stockfish-related repositories here

### DISCUSSION THREAD

**Discussion by dav1312** [official-stockfish/Stockfish#4013] [Ideas]
There are currently a lot of Stockfish-related repositories that are hard to find (or even know that they exist) because they are not in the organization
e.g.
![image](https://user-images.githubusercontent.com/63931154/167249753-bc27405b-9be4-4351-b89a-9191b2e4bcdb.png)

- [x] https://github.com/glinscott/nnue-pytorch
- [x] https://github.com/glinscott/fishtest
- [ ] https://github.com/ppigazzini/worker-setup
- [x] https://github.com/daylen/stockfish-web
- [ ] https://github.com/daylen/stockfish-mac
- [ ] https://github.com/vondele/matetrack
- [x] https://github.com/vondele/WLD_model

---

Docs about transferring a repository: https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository
> All links to the previous repository location are automatically redirected to the new location.

**Comment by pschneider1968** [comment]
It would also be nice if the Stockfish repo got a Wiki page of it's own, and if the Fishtest Wiki pages that document how to build Stockfish would be moved to it. Currently, all is a little bit spread out, and not easy to find for starters.

**Comment by ppigazzini** [comment]
Candidate to join the list https://github.com/ppigazzini/worker-setup

**Comment by Sopel97** [comment]
I'd like to see at least fishtest and nnue-pytorch to be moved. Though I'm not sure how many things would break after such a migration, as not everything supports redirects. I assume that the old repos would cease to exist and instead turn into redirects, as otherwise we would risk a mess with old references continuing to exist and silently cause havoc.

**Comment by ppigazzini** [comment]
The transfer seems reference safe according to the doc
https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository#whats-transferred-with-a-repository

It seems that the Leela repo was copied/moved/transferred to the Organization, so someone from Leela could have some experience to share.
https://github.com/glinscott/leela-chess

**Reply by ianfab** [reply]
To my understanding the only risk is that the original owner creates a repository with the same name, which can e.g. happen if they create a fork to file a PR after the transfer, and would by that override the reference/redirect. Github docs explicitly mention this risk for [organization renaming](https://docs.github.com/en/organizations/managing-organization-settings/renaming-an-organization#changes-that-occur-automatically), so I assume (but am not sure) that this also applies to the scenario of repository transfer.
